pkg/security/tests/process_test.go

```diff
@@ -661,45 +661,59 @@ func TestProcessLineage(t *testing.T) {
 	}
 	defer test.Close()
 
-	var execPid int
-	var lastEvent model.EventType
+	var execPid int // will be set by the first fork event
 
 	err = test.GetProbeEvent(func() error {
 		cmd := exec.Command(executable, "-t", "01010101", "/dev/null")
 		return cmd.Run()
 	}, func(event *sprobe.Event) bool {
+		if execPid != 0 && int(event.ProcessContext.Pid) != execPid {
```

Comment:
It seems to be possible to have an event outside out the testsuite matching this statement ? I would rather put a test in the Exit case, checking that we have an Exit event for the execPid

Reply:
Good point, I will change that